admin/app/controllers/admin/matches_controller.rb

```diff
@@ -4,6 +4,16 @@ module Admin
   class MatchesController < ApplicationController
     include Godmin::Resources::ResourceController
 
+    def conclude
+      Match.find(params[:id]).conclude
+      redirect_to resource, notice: "Matchingen avslutades"
+    end
+
+    def reactivate
+      Match.find(params[:id]).reactivate
+      redirect_to resource, notice: "Matchingens avslut ångrades"
```

Comment:
maybe we should be using the redirect_after_update path. And put all language in one of the ymls...?